src/tests/ftest/launch.py

```diff
@@ -885,32 +907,104 @@ def clean_logs(test_yaml, args):
     return True
 
 
-def archive_logs(avocado_logs_dir, test_yaml, args):
-    """Copy all of the host test log files to the avocado results directory.
+def get_remote_file_command():
+    """Get path to get_remote_files.sh script."""
+    return "{}/get_remote_files.sh".format(os.path.abspath(os.getcwd()))
+
+
+def compress_log_files(avocado_logs_dir):
+    """Compress log files.
 
     Args:
         avocado_logs_dir (str): path to the avocado log files
-        test_yaml (str): yaml file containing host names
+    """
+    print("Compressing files in {}".format(socket.gethostname().split(".")[0]))
+    logs_dir = os.path.join(avocado_logs_dir, "latest", "daos_logs", "*.log*")
+    command = [
+        get_remote_file_command(), "-z", "-x", "-v", "-f {}".format(logs_dir)]
```

Comment:
With the addition of the trap in `get_remote_files.sh`, we shouldn't need the verbose option set by default, for example:
```
[2020-10-14T08:32:39.497Z] Running /usr/lib/daos/TESTING/ftest/get_remote_files.sh -z -x -v -f /var/tmp/ftest/avocado/job-results/latest/daos_logs/*.log*
[2020-10-14T08:32:39.497Z] + trap 'if [ -f "${log_file}" ]; then cat "${log_file}"; fi' ERR
[2020-10-14T08:32:39.497Z] + check_files_input ' /var/tmp/ftest/avocado/job-results/latest/daos_logs/*.log*'
[2020-10-14T08:32:39.497Z] + '[' -n ' /var/tmp/ftest/avocado/job-results/latest/daos_logs/*.log*' ']'
[2020-10-14T08:32:39.497Z] + ls -d '/var/tmp/ftest/avocado/job-results/latest/daos_logs/*.log*'
[2020-10-14T08:32:39.497Z] + echo 'No files matched  /var/tmp/ftest/avocado/job-results/latest/daos_logs/*.log*. Nothing to do.'
[2020-10-14T08:32:39.497Z] No files matched  /var/tmp/ftest/avocado/job-results/latest/daos_logs/*.log*. Nothing to do.
[2020-10-14T08:32:39.497Z] + exit 0
```
 It should be set based upon the `args.verbose` setting:
```suggestion
    command = [
        get_remote_file_command(), "-z", "-x", "-f {}".format(logs_dir)]
    if args.verbose:
        command.append("-v")
```
Which will require passing `args` to this method.

Reply:
yes, right. Will change

Reply:
I see that the verbose flag is no longer included by default with the `get_remote_files.sh` command - which is good.  Is there a reason why we don't want to all `launch.py -v` to include using  `get_remote_files.sh -v` here like it is done in the other calls?